resources/language/resource.language.es_es/strings.po

```diff
@@ -524,7 +527,7 @@ msgstr "Purgar caché en memoria y disco"
 
 msgctxt "#30134"
 msgid "Enable execution timing"
-msgstr "Habilitar temporización de la ejecución"
+msgstr "Habilitar tiempo de ejecución"
 
```

Comment:
DON'T AGREE because I think the alternative name is not clear for this function. My approach is not perfect, but I think it is clearer for the average user. However, @CastagnaIT  I think you should rethink the name of this feature to be understandable for everyone.